src/core/config/descriptors/parameter_descriptor.cpp

```
@@ -226,6 +253,21 @@ std::experimental::any parameter_descriptor::get_value(const std::string &val) c
     throw std::experimental::bad_any_cast();
 }
 
+std::string parameter_descriptor::convert_int64_to_mapped_string(int64_t val) const
+{
+    if (m_string_mapping.empty()) {
+        return std::string();
+    }
+    
+    for (const auto &mapping : m_string_mapping) {
+        if ((mapping.second.type() == typeid(int64_t)) &&
+            (std::experimental::any_cast<int64_t>(mapping.second) == val)) {
+            return mapping.first;
+        }
+    }
+    return std::to_string(val) + " (Invalid value)";
```

do we want to throw an exception here? as this is an exceptional case?